Kitodo/src/main/java/org/kitodo/production/services/dataeditor/DataEditorService.java

```diff
@@ -143,13 +131,14 @@ public static String getTitleValue(LogicalDivision element, String metadataTitle
                     .flatMap(metadataGroup -> metadataGroup.getMetadata().stream())
                     .collect(Collectors.toList());
         }
-        Optional<String> metadataTitle = metadata.stream()
+        String metadataTitle = metadata.stream()
```

Comment:
Could the result of this method call not be returned directly? I am not sure if storing it to a separate variable for just one line has any benifit.

Reply:
Yes, done.